common/formatting/token_partition_tree.cc

```diff
@@ -444,6 +444,24 @@ static bool AppendFittingSubpartitions(
   return wrapped_first_subpartition;
 }
 
+// Update grouped childrens indentation in case of expanding grouping partitions
+static void UpdateGroupedChildrensIndentation(TokenPartitionTree* tree_node) {
+  for (auto& group : tree_node->Children()) {
+    const auto& group_uwline = group.Value();
+    const auto group_indent = group_uwline.IndentationSpaces();
+
+    for (auto& subpart : group.Children()) {
+      auto& subpart_uwline = subpart.Value();
+      const auto subpart_indent = group_indent - subpart_uwline.IndentationSpaces();
+
+      subpart.ApplyPreOrder([&subpart_indent](TokenPartitionTree& node) {
+        auto& uwline = node.Value();
+        uwline.SetIndentationSpaces(subpart_indent + uwline.IndentationSpaces());
+      });
+    }
+  }
+}
```

Comment:
You can use:

https://github.com/google/verible/blob/0b0011b50b7b9774b11d6e824ada0ff2f34ef6a4/common/formatting/token_partition_tree.cc#L137

or 

https://github.com/google/verible/blob/0b0011b50b7b9774b11d6e824ada0ff2f34ef6a4/common/formatting/token_partition_tree.cc#L144

to shift an entire subtree.

Reply:
Done. Thanks for quick respone!